active/progress/ProgressRing.md

```diff
@@ -69,73 +25,16 @@ The following examples show how to use the IsIndeterminate property to change th
 <ProgressRing IsActive="True" Height="100" Width="100"/>
 ```
 
-![](images/ProgressRing-indeterminate.PNG)
+![](images/ProgressRing-indeterminate.jpg)
 ## Determinate ProgressRing
 
 ```xml
 <ProgressRing IsActive="True" Height="100" Width="100" Value="75" IsIndeterminate="False"/>
```

Comment:
Default values for Minimum and Maximum from RangeBase are 0 and 1, so in this example value should be 0.75 right ?

Reply:
At least in Karen's implementations the progress ring style updates the values to 0 and 100 so this code would work as expected.  Open question of if its correct for the maximum to default to 100 or if we should leave it at 1

Reply:
Yes, I added an extra setter on `Maximum` property so that the default is 100. I believe this mirrors the default for ProgressBar.

Reply:
Yes I think the default for progress ring should mirror progress bar and be set to 100.

Reply:
@ranjesh do you agree with that?

Reply:
@ranjeshj are you ok with this?

Reply:
If ProgressBar already does this, then it makes sense to keep the two consistent.